cli/cmd/sync.go

```diff
@@ -65,8 +66,18 @@ func sync(cmd *cobra.Command, args []string) error {
 }
 
 func syncConnection(ctx context.Context, sourceSpec specs.Source, destinationsSpecs []specs.Destination) error {
+	destinationNames := make([]string, len(destinationsSpecs))
+	for i := range destinationsSpecs {
+		destinationNames[i] = destinationsSpecs[i].Name
+	}
 	syncTime := time.Now().UTC()
-	sourceClient, err := clients.NewSourceClient(ctx, sourceSpec.Registry, sourceSpec.Path, sourceSpec.Version)
+
+	log.Info().Str("source", sourceSpec.Name).Strs("destinations", destinationNames).Time("syncTime", syncTime).Msg("Start sync")
```

Comment:
nit: I think all our log args are snake_case, so it should apply to `sync_time` as well

Reply:
Updated